### PR TITLE
Store corruption fixes

### PR DESCRIPTION
**Pull request by K900**
systemd tries to create the autovt@ unit on startup. Make it not do that, and also make it so we can't fuck it up again.

### CODE REVIEW

File: scripts/syschdemd.nix [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    coreutils
+    daemonize
+    glibc  # getent
+    gnugrep
+    systemd  # machinectl
+    util-linux  # nsenter
+    wrapper
+  ];
+  inherit defaultUser defaultUserHome;
+}
```
Comment (SuperSandro2000) [inline scripts/syschdemd.nix:54]: Missing a final new line which is important that git can generate diffs properly.

File: scripts/syschdemd.nix [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  coreutils, 
+  daemonize,
+  glibc,
+  gnugrep,
+  systemd, 
+  util-linux, 
+  
+  defaultUser,
+  defaultUserHome
+}:
```
Comment (SuperSandro2000) [inline scripts/syschdemd.nix:15]: Can we use nixpkgs-fmt instead of alejandra?

File: scripts/syschdemd.nix [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+let
+  writeShellScript = { name, src, path, ... }@args:
+    stdenvNoCC.mkDerivation ({
```
Comment (SuperSandro2000) [inline scripts/syschdemd.nix:18]: This could be replaced with runCommand

File: scripts/wrapper.sh [hunk 4]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+mount -t binfmt_misc binfmt_misc /proc/sys/fs/binfmt_misc
+
+exec systemd
```
Comment (SuperSandro2000) [inline scripts/wrapper.sh:6]: also missing a final new line

File: scripts/syschdemd.nix (hunk 2 above)
Comment (K900) [reply scripts/syschdemd.nix:15]: This is actually formatted manually

**Pull request closed without merging**